App.js

```diff
@@ -115,6 +115,16 @@ const App = () => {
 
   appUpgradeVersionCheck(appInfo, APP_UPGRADE_API_KEY, alertConfig);
 
+  // Remove this method to stop OneSignal Debugging
+  OneSignal.Debug.setLogLevel(LogLevel.Verbose);
```

Comment:
Is it okay to put verbose logs in production? I think this will generate an extensive amount of detailed log messages. CMIIW

Reply:
mas @arilukmanp done on latest commit

Reply:
mas @arilukmanp done on latest commit